Gemfile

```
@@ -126,6 +126,9 @@ gem("mail")
 # for detecting file type of uploaded images
 gem("mimemagic")
 
+# for monitoring
+gem("newrelic_rpm")
+
```

I suggest moving to the end of the file. But more important, put it in a production group. Otherwise it will tryh to talk to the web in the test environment.
```suggestion
group :production do
  # Application performance monitoring
  # https://one.newrelic.com/marketplace
  gem('newrelic_rpm')
end
```